pynblint/nb_linting.py

```diff
@@ -152,6 +152,15 @@ def missing_closing_MD_text(notebook: Notebook) -> bool:
     )
 
 
+def not_renamed_duplicate_notebook(notebook: Notebook) -> bool:
+    """Check if the duplicate notebook has not been renamed.
+
+    I.e., if it was left with the default title: ``notebook-Copy.ipynb``.
+    """
+
+    return "-Copy" in notebook.path.name
```

Comment:
This is not general enough.

Think about the following possible renamings of a duplicate notebook:

- `Training-Copy-23-03-2022.ipynb`
- `FeatEngineering-Backup-Copy.ipynb`

In both cases, the substring `-Copy` is used on purpose.

I'd rather use a regex to solve this problem by taking into account only the case in which the pattern `-Copy<copy-number>` immediately precedes the notebook extension (`.ipynb`).

The following should do: `.*-Copy\d+.ipynb`. You might use it like so:

```python
import re

pattern = re.compile(r".*-Copy\d+.ipynb")
return pattern.match(notebook.path.name)
```

Reply:
I hadn't thought of such an eventuality. I followed your suggestion and now there should be no problems.